shared/src/mobileMain/kotlin/org/jetbrains/kotlinconf/ui/CodeOfConductScreen.kt

```diff
@@ -33,24 +27,11 @@ fun CodeOfConductScreen(back: () -> Unit) {
             onLeftClick = back,
             isRightVisible = false
         )
-        Column(
-            Modifier
-                .fillMaxWidth()
-                .background(MaterialTheme.colors.grey5Black)
-        ) {
-            Text(
-                "KotlinConf code of conduct",
-                style = MaterialTheme.typography.h2,
-                modifier = Modifier.padding(16.dp)
-            )
-            HDivider()
-        }
-        Column(
-            Modifier
-                .padding(16.dp)
-                .verticalScroll(rememberScrollState())
-        ) {
-            MarkdownFileView("files/code-of-conduct.md")
-        }
+        MarkdownScreenWithTitle(
+            "Code of conduct",
```

Comment:
Don't you want to extract it to the strings resources?

Reply:
done